GVRf/tools/Framework/settings.gradle

```diff
@@ -1,19 +1,19 @@
 include ':framework'
-if (!hasProperty("ARM64")) {
-    include ':backend_oculus'
-}
+//if (!hasProperty("ARM64")) {
```

Comment:
Yes. This is for the experimental workspace and the 64bit support there is not a priority.